packages/cli/openapi-ir-to-fern/src/converters/convertEndpoint.ts

```diff
@@ -226,13 +228,136 @@ export function convertEndpoint({
+        return convertEndpointExample(endpointExample);
+    });
+}
+
+function convertEndpointExample(endpointExample: EndpointExample): RawSchemas.ExampleEndpointCallSchema {
+    return {
+        "path-parameters":
+            endpointExample.pathParameters != null && endpointExample.pathParameters.length > 0
+                ? convertNamedFullExamplesToExampleTypeReferenceSchemas(endpointExample.pathParameters)
+                : undefined,
+        "query-parameters":
+            endpointExample.queryParameters != null && endpointExample.queryParameters.length > 0
+                ? convertNamedFullExamplesToExampleTypeReferenceSchemas(endpointExample.queryParameters)
+                : undefined,
+        headers:
+            endpointExample.headers != null && endpointExample.headers.length > 0
+                ? convertNamedFullExamplesToExampleTypeReferenceSchemas(endpointExample.headers)
+                : undefined,
+        request:
+            endpointExample.request != null
+                ? convertFullExampleToExampleTypeReferenceSchema(endpointExample.request)
+                : undefined,
+        response:
+            endpointExample.response != null
+                ? { body: convertFullExampleToExampleTypeReferenceSchema(endpointExample.response) }
+                : undefined,
+    };
+}
+
+function convertNamedFullExamplesToExampleTypeReferenceSchemas(
+    namedFullExamples: NamedFullExample[]
+): Record<string, RawSchemas.ExampleTypeReferenceSchema> {
+    const result: Record<string, RawSchemas.ExampleTypeReferenceSchema> = {};
+    namedFullExamples.map(
+        (namedFullExample) =>
+            (result[namedFullExample.name] = convertFullExampleToExampleTypeReferenceSchema(namedFullExample.value))
+    );
+    return result;
+}
+
+function convertFullExampleToExampleTypeReferenceSchema(
+    fullExample: FullExample
+): RawSchemas.ExampleTypeReferenceSchema {
+    return FullExample._visit<RawSchemas.ExampleTypeReferenceSchema>(fullExample, {
+        primitive: (primitive) => convertPrimitiveExampleToExampleTypeReferenceSchema(primitive),
+        object: (object) => convertFullExamplePropertiesToExampleTypeReferenceSchema(object.properties),
+        array: (array) => convertFullArrayExampleToExampleTypeReferenceSchema(array),
+        map: (map) => convertFullMapExampleToExampleTypeReferenceSchema(map),
+        oneOf: (oneOf) => convertFullOneOfExampleToExampleTypeReferenceSchema(oneOf),
+        enum: (value) => value,
+        literal: (value) => value,
+        unknown: (unknown) => convertFullExampleToExampleTypeReferenceSchema(unknown),
+        _unknown: () => "unknown",
+    });
+}
+
+function convertPrimitiveExampleToExampleTypeReferenceSchema(
+    primitiveExample: PrimitiveExample
+): RawSchemas.ExampleTypeReferenceSchema {
+    return PrimitiveExample._visit<RawSchemas.ExampleTypeReferenceSchema>(primitiveExample, {
+        int: (value: number) => value.toString(),
```

Comment:
hmm does this actually produce valid yaml, i think you want this to an int

Reply:
```yaml
example: 
  myNumber: 9
  myNumber: "9" 
```

Reply:
Good catch, will update all the number-like cases (and boolean).